src/components/WhaleSightingViewer/SightingList.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { getWhaleSightings, NewSighting } from "../../clients/apiClient";
+import "./SightingList.scss";
+
+export default function SightingList(props: {
+    pageNum: number,
+    sightingsList: NewSighting[],
+}) { 
+
+    if (!props.sightingsList) return <p>Waiting for data...</p>
+
+    const sightingsToShow = Math.min(props.sightingsList.length, ((props.pageNum-1) * 12 + 12));
```

Comment:
Is this the number of sightings we will show? Could we rename it to be more explicit?